pkgs/by-name/ar/arpwatch/package.nix

```diff
@@ -0,0 +1,38 @@
+{ fetchurl
+, lib
+, libpcap
+, stdenv
+, system-sendmail
+, ...
+}:
+stdenv.mkDerivation rec {
+  pname = "arpwatch";
+  version = "3.6";
+
+  src = fetchurl {
+    url = "https://ee.lbl.gov/downloads/${pname}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-+GUp/lf9taL/VBO8E8JFBj+Zs790JCH9MTMnIXW+gVY=";
+  };
+
+  patchPhase = ''
+    sed -i '1i#include <time.h>' report.c
+  '';
+
+  preInstall = ''
+    install -d -m 0755 $out/bin
+    install -d -m 0755 $out/etc/rc.d
+    install -d -m 0755 $out/share/man/man8
+  '';
+
+  configureFlags = [ "--sbindir=${placeholder "out"}/bin" ];
```

Comment:
The resulting `/nix/store/9c5cvz3pz8fpcnh11ny2k2r2w8a3v1qb-arpwatch-3.6/etc/rc.d/arpwatch` file seems to reference two directories that don't exist:
- `/nix/store/9c5cvz3pz8fpcnh11ny2k2r2w8a3v1qb-arpwatch-3.6/sbin` (line 14)
- `/nix/store/9c5cvz3pz8fpcnh11ny2k2r2w8a3v1qb-arpwatch-3.6/arpwatch` (line 17)